Migration to VCD (#6625)
ovh/docs: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd_migration_use-cases/guide.fr-fr.md

---
title: 'VMware Cloud Director - Audit des cas particuliers de migration vers VCD'
excerpt: 'Découvrez les méthodes d'examen des scénarios difficiles au sein de vos services vSphere/vCenter managé OVHcloud, en préparation à une migration vers VCD'

> updated: 2024-07-22

Reviewer: ```suggestion
updated: 2024-07-23
```

---
title: 'VMware Cloud Director - Audit des cas particuliers de migration vers VCD'
excerpt: 'Découvrez les méthodes d'examen des scénarios difficiles au sein de vos services vSphere/vCenter managé OVHcloud, en préparation à une migration vers VCD'

> updated: 2024-07-19

Reviewer: ```suggestion
updated: 2024-07-22
```

Author: Hello @Kazarus4, wouldn't it be easier to commit instead of comment, I review them of course ?

| **Étape 2**  | ?           | ❎                 | **FT (tolérance aux pannes)**               | **Pas de tolérance aux pannes.**                                                                                                                    | - Mon service vSphere managé chez OVHcloud a-t-il des VM avec FT (Fault Tolerance) activé ?                                                                                                      | [VMware Fault Tolerance](/pages/bare_metal_cloud/managed_bare_metal/vmware_fault_tolerance).                                                                                         |

> | **Étape 3**  | ?           | ❎                 | **Affinité/anti-affinité DRS**              | **Aucune règle d’affinité/anti-affinité DRS.**                                                                                                      | - La migration de VMs à VMs peut être migrée vers VCD, mais de VM à hôtes ne peut pas être migrée.                                                                                               | [VMware DRS (Distributed resource Scheduler](/pages/bare_metal_cloud/managed_bare_metal/vmware_drs_distributed_resource_scheduler).                                                  |

Reviewer: ```suggestion
| **Étape 3**  | ?           | ❎                 | **Affinité/anti-affinité DRS**              | **Aucune règle d’affinité/anti-affinité DRS.**                                                                                                      | - La migration de VM à VM est possible vers VCD, mais pas de VM à hôtes.                                                                                               | [VMware DRS (Distributed resource Scheduler](/pages/bare_metal_cloud/managed_bare_metal/vmware_drs_distributed_resource_scheduler).                                                  |
```

> | **Étape 4**  | ?           | ❎                 | **Périphériques spéciaux (CD, DVD, etc.)**  | **Aucun équipement spécial ne doit être branché au datacenter géré par vSphere migré.**                                                             | - Tous les périphériques spéciaux (CD, DVD, etc.) doivent être retirés avant la migration, sinon ils seront retirés de force par le processus de migration et perdus.                            |                                                                                                                                                                                      |

Reviewer: ```suggestion
| **Étape 4**  | ?           | ❎                 | **Périphériques spéciaux (CD, DVD, etc.)**  | **Aucun équipement spécial ne doit être branché au datacenter géré par le vSphere migré.**                                                             | - Tous les périphériques spéciaux (CD, DVD, etc.) doivent être retirés avant la migration. Sinon, ils seront retirés de force par le processus de migration et perdus.                            |                                                                                                                                                                                      |
```

> | **Étape 5**  | ?           | ❎                 | **Clusters de datastores**                  | **Assurez-vous que toutes les règles de clustering sont supprimées dans vSphere sur OVHcloud.**                                                     | - Les règles de clustering devront être supprimés avant la migration car cette notion n'existe pas côté VCD.                                                                                     |                                                                                                                                                                                      |

Reviewer: ```suggestion
| **Étape 5**  | ?           | ❎                 | **Clusters de datastores**                  | **Assurez-vous que toutes les règles de clustering sont supprimées dans vSphere on OVHcloud.**                                                     | - Les règles de clustering devront être supprimés avant la migration car cette notion n'existe pas côté VCD.                                                                                     |                                                                                                                                                                                      |
```

> | **Étape 6**  | ?           | ❎                 | **Sur-engagement mémoire**                  | **Scale up VCD.**                                                                                                                                   | - Prévoyez plus de ressources côté VCD ou redimensionnez/optimisez les VM avant la migration côté vSphere. <br/> - Parce que vous ne pourrez pas sur-engager (over-commit) des ressources.       | [Modification des ressources de la machine virtuelle](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/modify_hardware_configuration_of_vm).                       |

Reviewer: ```suggestion
| **Étape 6**  | ?           | ❎                 | **Sur-engagement mémoire**                  | **Scale up VCD.**                                                                                                                                   | - Prévoyez plus de ressources côté VCD ou redimensionnez/optimisez les VM avant la migration côté vSphere. <br/> - Vous ne pourrez pas sur-engager (over-commit) des ressources.       | [Modification des ressources de la machine virtuelle](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/modify_hardware_configuration_of_vm).                       |
```

| **Étape 8**  | ?           | ❎                 | **Options de sécurité (PCI-DSS, HDS, SNC)** | **Ne pas avoir de vSphere certifiés PCI-DSS, HDS, SNC.**                                                                                            | - Ne peut pas être migré si le vSphere managé on OVHcloud est certifié PCI-DSS, HDS, SNC (vSphere avec options de sécurité).                                                                     |                                                                                                                                                                                      |

> | **Étape 9**  | ?           | ❌                 | **Encrypted VMs**                           | **Déchiffrer les VM dans vSphere on OVHcloud.**                                                                                                     | - Impossible de migrer avec des données (VMs, vApp)chiffrées vSphere managé on OVHcloud.                                                                                                         |                                                                                                                                                                                      |

Reviewer: ```suggestion
| **Étape 9**  | ?           | ❌                 | **Encrypted VMs**                           | **Déchiffrer les VM dans vSphere on OVHcloud.**                                                                                                     | - Impossible de migrer avec des données (VMs, vApp) chiffrées vSphere managé on OVHcloud.                                                                                                         |                                                                                                                                                                                      |
```

> | **Étape 10** | ?           | ❌                 | **Zerto**                                   | **No Zerto.**                                                                                                                                       | - Si vous utilisez Zerto, vous ne pouvez pas migrer vers VCD pour le moment.                                                                                                                     | [Setting up Zerto Virtual Replication between two OVHcloud data centers](/pages/hosted_private_cloud/hosted_private_cloud_powered by_vmware/zerto_virtual_replication_as_a_service). |

Reviewer: ```suggestion
| **Étape 10** | ?           | ❌                 | **Zerto**                                   | **No Zerto.**                                                                                                                                       | - Si vous utilisez Zerto, vous ne pouvez pas migrer vers VCD pour le moment.                                                                                                                     | [Mise en place de Zerto Virtual Replication entre deux centres de données OVHcloud](/pages/hosted_private_cloud/hosted_private_cloud_powered by_vmware/zerto_virtual_replication_as_a_service). |
```

> | **Étape 11** | ?           | ❎                 | **Hosts / Zpool**                           | **Libérez des ressources (hôtes + Zpool) gratuitement et toutes les heures avant la migration. <br/> Ou convertissez-le en ressources mensuelles.** | - Libérez les ressources (hôtes + Zpool) "freespare" et "hourly" avant la migration. <br/> Ou convertissez-le en ressources "monthly".                                                           | [Informations de facturation du Hosted Private Cloud](/pages/account_and_service_management/manage_billing_payment_and_services/facturation_private_cloud).                          |

Reviewer: ```suggestion
| **Étape 11** | ?           | ❎                 | **Hosts / Zpool**                           | **Libérez des ressources (hôtes + Zpool) gratuitement et toutes les heures avant la migration. <br/> Ou convertissez-le en ressources mensuelles.**                                                         | [Informations de facturation du Hosted Private Cloud](/pages/account_and_service_management/manage_billing_payment_and_services/facturation_private_cloud).                          |
```

Reviewer: must be check translation

> | **Étape 12** | ?           | ❌                 | **HCX**                                     | **Pas d’utilisation HCX dans vSphere sur OVHcloud.**                                                                                                | - Si tel est le cas, il ne peut pas être migré vers VCD.                                                                                                                                         |                                                                                                                                                                                      |

Reviewer: ```suggestion
| **Étape 12** | ?           | ❌                 | **HCX**                                     | **Pas d’utilisation HCX dans vSphere sur OVHcloud.**                                                                                                | - Si tel est le cas, imposible de migrer vers VCD.                                                                                                                                         |                                                                                                                                                                                      |
```
